ui/ozone/platform/wayland/fake_server.cc

```diff
@@ -28,6 +29,7 @@ constexpr uint32_t kOutputVersion = 2;
 constexpr uint32_t kDataDeviceManagerVersion = 3;
 constexpr uint32_t kSeatVersion = 4;
 constexpr uint32_t kTextInputManagerVersion = 1;
+constexpr uint32_t kLinuxDmabufVersion = 1;
```

Comment:
Is there a particular reason why kLinuxDmabufVersion = 1 is used? WaylandConnection binds up-to version 3 of the zwp dmabuf linux global object.